### DISCUSSION TITLE
Integration with REST APIs

### DISCUSSION THREAD

**Discussion by fpandolfi64** [globaleaks/globaleaks-whistleblowing-software#3357] [General]
HI,
I would like to know if it is possible to integrate Globlaleaks with REST API to build questionnaires dynamically. For example, populate a selection box with the result of a json.

Thank you

**Comment by evilaliv3** [comment]
Thank you for opening this discussion @fpandolfi64 

At the moment it is not possible but it is definitely something interesting as a further extent.

do you have in mind a specific standard format ? Please feel free to move forward your ideas and eventually if you are a developer to propose specific changes.

**Reply by fpandolfi64** [reply]
Thanks for the reply. I try to develop my idea and propose it. Can I always use this forum to post it?

**Comment by evilaliv3** [comment]
Yes please!

If you will be producing already a detailed analysis useful to move forward
the development you can also write directly a technical ticket here:
https://github.com/globaleaks/GlobaLeaks/issues

I invite you to explore if there are already other tickets that discuss
similar ideas
